core/settings.py

```diff
@@ -168,3 +169,16 @@
         },
     },
 }
+
+# Sentry Configuration
+sentry_sdk.init(
+    dsn=os.environ.get("SENTRY_DSN"),
+    # Set traces_sample_rate to 1.0 to capture 100%
+    # of transactions for performance monitoring.
+    traces_sample_rate=1.0,
+    # Set profiles_sample_rate to 1.0 to profile 100%
+    # of sampled transactions.
+    # We recommend adjusting this value in production.
```

Comment:
Can you allow these to be overriden by environment variables so we can adjust for prod later?

Reply:
Yes, I am happy to change this when required for Prod.